tomcat/src/main/java/org/apache/coyote/http11/controller/RegisterController.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.coyote.http11.controller;
+
+import nextstep.jwp.model.Request;
+import org.apache.coyote.http11.service.RegisterService;
+import nextstep.jwp.vo.FileName;
+import nextstep.jwp.vo.FormData;
+import nextstep.jwp.vo.Response;
+
+import java.io.IOException;
+
+public class RegisterController implements Controller {
+
+    private static final RegisterController INSTANCE = new RegisterController();
+
+    @Override
+    public Response respond(Request request) throws IOException {
+        FileName fileName = request.getFileName();
+        FormData requestBody = request.getBody();
```

Comment:
```suggestion
    public Response respond(Request request) {
        FormData requestBody = request.getBody();
```
여기도 마찬가지로 fileName과 IOException이 사용되지 않고 있어요

Reply:
수정하였습니다